pkg/operator/controllers/cpms/cpms_controller.go

```diff
@@ -0,0 +1,100 @@
+package cpms
+
+import (
+	"context"
+
+	machinev1 "github.com/openshift/api/machine/v1"
+	"github.com/sirupsen/logrus"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+
+	"github.com/Azure/ARO-RP/pkg/operator"
+	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
+	"github.com/Azure/ARO-RP/pkg/operator/controllers/base"
+)
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+const (
+	ControllerName         = "CPMSController"
```

Comment:
nit: similar to my comments below, I think `CPMSDisablerController` is a better name, more future-proof to when we use CPMS across the fleet (if that happens)

Reply:
We plan to use this controller to operationalize CPMS once we're ready for it.